bitsandbytes/cuda_setup/main.py

```diff
@@ -28,16 +28,15 @@
 
 from .env_vars import get_potentially_lib_path_containing_env_vars
 
-if platform.system() == 'Windows':  # Windows
+DYNAMIC_LIBRARY_SUFFIX = { "Darwin": ".dylib", "Windows": ".dll", "Linux": ".so"}.get(platform.system(), ".so")
+if DYNAMIC_LIBRARY_SUFFIX == ".dll":  # Windows
```

Comment:
I think
```suggestion

if platform.system() == 'Windows':  # Windows
```
would still be clearer :)